### DISCUSSION TITLE
Git branch strategy

### DISCUSSION THREAD

**Discussion by security-companion** [security-companion/w3af#337] [General]
Hi,
we probably do not need this yet but probably after our first alpha-release.
What is the git branching strategy that we want to use for this project?
In the old w3af docu I found that w3af was using git flow, see eg. https://w4af.readthedocs.io/en/latest/update.html#branches
Is this something we want to keep on using for w4af?
I would suggest to leave it like it was used on the old w3af project to keep some consistency.
What do you think?

**Comment by codders** [comment]
I'm fine to use git flow once we are past the first alpha release. It's a good way for us to organise ourselves between releases.

I would also prefer if pull requests come from Github forks rather than creating branches in the main project, unless we really need to be able to collaborate on a pull request (which should happen too often).
